### PR TITLE
COVID-19Predict: Merging forecast starting at 2021-11-14

### PR DESCRIPTION
**Pull request by AustinMathuw**

### CODE REVIEW

**Comment by eycramer** [comment]
Hi @AustinMathuw, 

Can you please change your  team_name to: CWRU and your model_name to: COVID-19Predict

Thanks,
Estee

File: data-processed/CWRU-COVID_19Predict/metadata-CWRU-COVID_19Predict.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+team_name: COVID-19Predict
```
Comment (elray1) [inline data-processed/CWRU-COVID_19Predict/metadata-CWRU-COVID_19Predict.txt:1]:
```suggestion
team_name: CWRU
```

File: data-processed/CWRU-COVID_19Predict/metadata-CWRU-COVID_19Predict.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+model_name: Dynamic 7-day Short-Term Predictions
```
Comment (elray1) [inline data-processed/CWRU-COVID_19Predict/metadata-CWRU-COVID_19Predict.txt:2]:
```suggestion
model_name: COVID_19Predict
```

**Review by elray1 — CHANGES_REQUESTED** [review]

**Review by elray1 — APPROVED** [review]

**Comment by AustinMathuw** [comment]
Thank you @elray1 for doing the changes!

**Pull request merged**